project/Dependencies.scala

```diff
@@ -21,6 +21,7 @@ object Dependencies {
     // Akka is provided because Spark already includes it, and Spark's version is shaded so it's not safe
     // to use this one
     "com.typesafe.akka" %% "akka-slf4j" % "2.3.4" % "provided",
+    "com.typesafe.akka" %% "akka-cluster" % "2.3.4",
```

Comment:
I'll exclude akka-remote, but this will only matter for Spark 1.5.x anyways, because Akka has been kicked out of spark 1.6.  :/